test/sanity/validate-modules/validate-modules

```diff
@@ -569,13 +570,20 @@ class ModuleValidator(Validator):
                             303,
                             'DOCUMENTATION fragment missing: %s' % fragment
                         ))
-                    except Exception as e:
-                        self.traces.append(e)
+                    except Exception:
+                        self.traces.append(traceback.format_exc())
                         self.errors.append((
                             304,
                             'Unknown DOCUMENTATION error, see TRACE'
                         ))
 
+                if 'options' in doc and doc['options'] is None and doc.get('extends_documentation_fragment'):
+                    self.errors.append((
+                        305,
```

Comment:
Should this have its own error code to differentiate it from the 305 error code below?